public/static/docs/command-reference/install.md

```diff
@@ -121,17 +120,15 @@ $ dvc pull --all-branches --all-tags
 ## Example: Checkout both DVC and Git
 
 Let's start our exploration with the impact of `dvc install` on the
-`dvc checkout` command. Remember that switching from one Git repository version
-to another (with `git checkout`) changes the set of
-[DVC-files](/doc/user-guide/dvc-file-format) in the project. This changes the
-set of data files that should be located in the workspace (which can be achieved
-with `dvc checkout`).
+`dvc checkout` command. Switching among <abbr>repository</abbr> versions (with
+`git checkout`) changes the set of [DVC-files](/doc/user-guide/dvc-file-format)
+in the project. This changes the set of data files that should be located in the
```

Comment:
`project directory` would be better

Reply:
Using "workspace" here. That + other updates in the file done in 296c2f04.